apis/Google.Logging.V2/Google.Logging.Log4Net/Extensions.cs

```diff
@@ -29,12 +29,16 @@ internal static class Extensions
 
```

Comment:
Removed both Timestamp and Scale extension methods, inlined at their call-sites. Scale() is for multiplying a timespan by a multiplier, for backoff.